core-web/apps/dotcms-ui/src/app/portlets/dot-containers/container-list/container-list.component.ts

```diff
@@ -105,7 +106,9 @@ export class ContainerListComponent implements OnDestroy {
      */
     updateSelectedContainers(containers: DotContainer[]): void {
         const filterContainers = containers.filter(
-            (container: DotContainer) => container.identifier !== 'SYSTEM_CONTAINER'
+            (container: DotContainer) =>
+                container.identifier !== 'SYSTEM_CONTAINER' &&
+                container.source !== CONTAINER_SOURCE.FILE
```

Comment:
I just got this news today, but hide the system container, we decided not to show it anymore because the user can't do anything.

I think you can update the endpoint url so it doesn't come back in the response.

Sorry for the back and forth on this, doing containers brings new questions.

Reply:
Noted

Reply:
Updated